diff --git a/base/BUILD.gn b/base/BUILD.gn
index cea72f13c4ce86701eb4fc9562afee2521f2e445..a860229fb8ef7fbe74785357f2ce8017ce373d38 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4416,6 +4416,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
