diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index c175e896868e99391ee8382d37fa08e17afb6768..66f8c7ee66e3190f39e2b9edd29630930fca3bf1 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -191,6 +191,7 @@
   X("webrtc")                                                            \
   X("webrtc_stats")                                                      \
   X("xr")                                                                \
+  BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES(X) /*see chromium_src*/   \
   X(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))                 \
   X(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))                      \
   X(TRACE_DISABLED_BY_DEFAULT("audio"))                                  \
