diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index e4e04f2e836af76ba75650d47cf6815444256249..ebe34a97bab3d73206009770e8eb478f29f9f6d4 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -733,6 +733,7 @@ class BASE_EXPORT [[maybe_unused, nodiscard]] ScopedAllowBaseSyncPrimitives {
   ScopedAllowBaseSyncPrimitives& operator=(
       const ScopedAllowBaseSyncPrimitives&) = delete;
 
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
