diff --git a/build/rust/cargo_crate.gni b/build/rust/cargo_crate.gni
index 427fcf69da0b5b15f972b58e3988c7d103bae80d..cdc9044bea9fe4c2f72e764b0c7603f3dcc008ae 100644
--- a/build/rust/cargo_crate.gni
+++ b/build/rust/cargo_crate.gni
@@ -214,7 +214,7 @@ template("cargo_crate") {
                                  "rustenv",
                                  "dev_deps",
                                ])
-    forward_variables_from(invoker, TESTONLY_AND_VISIBILITY)
+    forward_variables_from(invoker, [ "testonly" ])
     if (defined(crate_type) && crate_type == "cdylib") {
       # See comments above about cdylib.
       crate_type = "rlib"
