diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
index 3ca401a4337b8e4229021fe6faa0d30dbf25544c..e6a1f97d34a0a0e3d67f0cfb50a18e5e998bade1 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
@@ -101,6 +101,7 @@ export class ProfilePickerMainViewElement extends
     if (!isProfileCreationAllowed()) {
       this.$.addProfile.style.display = 'none';
     }
+    this.$.addProfile.style.borderRadius = '8px';
 
     this.addEventListener('view-enter-finish', this.onViewEnterFinish_);
 
