diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index edc513ebc5480c93fd87abf184e8e3783e231ad3..698a894b15d83d4caa484eb28227ccf9f7943e1a 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -97,6 +97,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -272,6 +273,7 @@ static_library("common_lib") {
   if (enable_ppapi) {
     public_deps += [ "//ppapi/shared_impl" ]
   }
+  import("//brave/common/sources.gni") sources += brave_common_sources
 
   if (enable_extensions) {
     sources += [
@@ -540,6 +542,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
