diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 6eedc814307cab742527ba1a3e6d835f06523072..08c575950f929eee46315c3ecb7d0582a41f9d62 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -439,6 +439,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
       "safetyCheckShowReviewPanel",
       base::FeatureList::IsEnabled(features::kSafetyCheckExtensions));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
