diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
index a7d852b6f481b22ac0555254f6e7ccdf3a7b4dbd..1702ec6123eca5c8a6f6b130a7aa85f7a4286821 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
@@ -192,6 +192,7 @@ PermissionPromptBubbleBaseView::PermissionPromptBubbleBaseView(
   }
 
   SetProperty(views::kElementIdentifierKey, kMainViewId);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_BASE_VIEW
 }
 
 PermissionPromptBubbleBaseView::~PermissionPromptBubbleBaseView() = default;
