diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 3c30781a83b182d3080d5e22b52b270dbfe99ea3..fab43421d4d491cc0b663eedb38e6f1025fbfae8 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -116,6 +116,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/ViewShiftingActionBarDelegate.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/VisibleUrlText.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
