diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 63ad4a7ea331a75fca3fc6ce53417e8622ff89b7..1ac8e709bbfc4b9a44fc49e4d03aaab66a1bbb95 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3831,6 +3831,7 @@ const FeatureEntry::FeatureVariation kPasswordGenerationExperimentVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
