diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 123ea468f1c32714dbbd5f1e074bb3c080701ba4..33e4a225438b34ce0ccf1c2a7c7d5f9fc3d0b9db 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3813,6 +3813,7 @@ const FeatureEntry::FeatureVariation kPasswordGenerationExperimentVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
