diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
index 1d6f10c9c34df0a976e98d6b553acf863fb3c4f6..8a96ee085b456d5e90bf6cce8da7c92e818c6b97 100644
--- a/chrome/installer/util/BUILD.gn
+++ b/chrome/installer/util/BUILD.gn
@@ -205,6 +205,7 @@ static_library("with_no_strings") {
       "initial_preferences.h",
     ]
   }
+  import("//brave/chromium_src/chrome/installer/util/sources.gni") deps += brave_chromium_src_chrome_installer_util_deps sources += brave_chromium_src_chrome_installer_util_sources
 }
 
 # Use this version of installer_util to link to the generated strings in .rc
@@ -241,6 +242,7 @@ generate_embedded_i18n("generate_strings") {
   output_file_name_base = "installer_util_strings"
 
   branding = branding_path_product
+  sources = [ "//chrome/app/brave_strings.grd" ]
 }
 
 # Compile the generated .rc file.
