diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index c8d3aa8b054a285b6340f60d1e4fffe854070ae7..92e097aff12fadf3152eb1184dd9fa4dd163c5fa 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -117,6 +117,7 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
