diff --git a/chrome/common/extensions/api/developer_private.idl b/chrome/common/extensions/api/developer_private.idl
index 4dd8268a7977646781c60bc2b5d6f1757ac29ec7..ba1beaadf45dc90760809920b0688f6d49591426 100644
--- a/chrome/common/extensions/api/developer_private.idl
+++ b/chrome/common/extensions/api/developer_private.idl
@@ -245,6 +245,7 @@ namespace developerPrivate {
     DOMString iconUrl;
     DOMString id;
     AccessModifier incognitoAccess;
+    boolean isSplitMode;
     DOMString[] installWarnings;
     DOMString? launchUrl;
     Location location;
