diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 0176a1cc47264ceffa24621b3e095c4be4ce3d99..b66ee7fe88357a996217c4ee2fab0a5b7d89fc93 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -136,6 +136,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_system.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
