diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 38de1dfd041e44d868329b9c9f4502aec82a6543..0403fc123034880a2deb82e882475ce78ad937a9 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -323,6 +323,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   RequestPermissionsInternal(permissions_types, render_frame_host,
                              requesting_origin, user_gesture,
@@ -372,6 +373,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
