diff --git a/components/BUILD.gn b/components/BUILD.gn
index ea3ec6f74f477ae119de778a3dc22cdf7ec22947..184514468ac125cacf94e74bc3c5ac8cf97db6e3 100644
--- a/components/BUILD.gn
+++ b/components/BUILD.gn
@@ -774,7 +774,7 @@ repack("components_tests_pak") {
   if (is_chrome_branded) {
     sources += [ "${root_gen_dir}/components/strings/components_google_chrome_strings_en-US.pak" ]
   } else {
-    sources += [ "${root_gen_dir}/components/strings/components_chromium_strings_en-US.pak" ]
+    sources += [ "${root_gen_dir}/components/strings/components_${branding_path_product}_strings_en-US.pak" ]
   }
 }
 
