diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index aa252794c42f41c5a3bd04fad8230cb832292475..976fb3925fabc0b2ffd7fae5b77f5e5fcdee17b0 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -122,6 +122,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
