diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index ba8f95509ec2b713bc8150a70d15fc79cc7c3e3c..ca5d9736e537fa18269cfc682c5a2ddaf487f63d 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -248,6 +248,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
