diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index 01c34c47f678ddf4a6c0d5e958a0f86e71181829..c45ccb3c263b7a83aa77efa61dfd5b88a823cc2f 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -114,4 +114,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
