diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index a4f5ba9e3846793f80f0dde24bcd8ac126092aff..608e21c0ed21e6d83c1bc3bf6814656b58bcb6d8 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1230,6 +1230,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
