diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 945d6f9bea465016c65e5574ba317d7a0af7bbd2..bd3f9f56d3f62b2b59aa468070b1be34517dd6ef 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -54,6 +54,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
