diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 4d89822c5cabc1d487dbf19c07c0be77ef3604c4..d3f140834c60eba84a9485d098fc57602a9e17d9 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2200,6 +2200,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
