diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index e9cf82103ada26079aef19dee0c521cf09cc8f46..a382fb6e9391e5fbcaade24bcb8d6742623da9e9 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1212,6 +1212,7 @@ static_idl_files_in_modules = [
   "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
   "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
 ]
+static_idl_files_in_modules += brave_static_idl_files_in_modules
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
