diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index 17f7dbf016bc3e556a0ff56cb6ef46a887efb2b5..cc8aa7b452fd5015e1829c266c21a4ce532ac91a 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -63,6 +63,7 @@ build_webui("build") {
       web_component_files +=
           [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ]
     }
+    else if (!is_android) { web_component_files += [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ] }
 
     icons_html_files = [
       "cr_fingerprint/cr_fingerprint_icons.html",
