diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index e54af01a8f5040c027f234fe24499d564046242e..4f901edbb55049aec151ab2ec15b9ab668c85cab 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2965,6 +2965,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
